### ISSUE TITLE
Extension does not work

### ISSUE THREAD

**Issue by rubenmamo** [Mimeo/VSTSExtension-ActivePullRequests#5]
I installed the extension on our On Premise TFS server, however when I click "All Pull Requests" I get a **Active Pull Requests by Mimeo failed to load.** error

I have installed version 0.2.4 of the extension. Is there any logs I can look for to try to identify the problem?

TFS Installed on Windows Server 2012 R2
Client is Windows 8 with Internet Explorer 11 browser.

I get the following in the IE Console Window:

```
SCRIPT1002: Syntax error
File: hub.html, Line: 193, Column: 39
```

When I open hub.html the code part in question is

```
                window.setTimeout( () => {
                    prCount = activePullRequests.length;
                    if(prCount === 0) {
                        grid.setDataSource([{
                            id: "",
                            title: "",
                            repo: "",
                            vote: ""
                        }]);
                    } 
                    $("#hub-title").text($("#hub-title").text() + " (" + prCount + " pull requests)");
                }, 5000);
```

It seems to not like the () => part as the cursor goes there

**Comment by nshenoy** [comment]
Thanks for alerting me to the issue @rubenmamo . This is happening because IE11 doesn't support ES6 lambdas. Let me try to rework this a little to be backwards compatible.

**Comment by nshenoy** [comment]
This specific lambda issue appears to be fixed now in v 0.2.5. However, IE11 will now run into issue #8 where it's crapping out on not supporting the Array.find() method. I'll close this issue and we can track the new IE11 issue in #8. Please reopen this issue if you're still hitting it with the new version of the extension.
